cdci_data_analysis/analysis/parameters.py

```diff
@@ -164,13 +164,23 @@ def __init__(self,
                  value=None,
                  units=None,
                  name: Union[str, None] = None,
+                 # TODO should we remove units/type/format knowledge from the Parameter class?
+
```

Comment:
I suggest to add a docstring for this class. The structure starts to become convoluted.

```suggestion
format:
   every parameter has a format. Format defines representation of the parameter value. 
   Default format of  defines unique representation o
```